autoload/neomake/makers/ft/php.vim

```diff
@@ -27,9 +27,8 @@ endfunction
 function! neomake#makers#ft#php#phpcs() abort
     let l:args = ['--report=csv']
 
-    "Add standard argument if one is set.
-    if exists('g:neomake_php_phpcs_args_standard')
-        call add(l:args, '--standard=' . expand(g:neomake_php_phpcs_args_standard))
+    if exists('g:neomake_php_phpcs_args_post')
+        call add(l:args, , g:neomake_php_phpmd_args_post))
```

Comment:
That would require to use the `fn` constructor callback curently.
Related: https://github.com/neomake/neomake/issues/843#issuecomment-291951164.